src/main.rs

```diff
@@ -70,6 +71,9 @@ pub struct Opts {
     /// After filling in not-found licenses, check if new is a strict subset of previous.
     #[structopt(long, short)]
     check_previous: bool,
+
+    #[structopt(long, short)]
+    human_readable_artifact: bool,
```

Comment:
question: would you be opposed to adding `HumanReadable` as a `format::Format`?

And then we could avoid a special-case flag and handling for just outputing a text file.

Alternatively, I think adding a subcommand for this, or calling this flag `convert_to_human_readable_only` or something like that to make it clear what this is doing.